src/common/roll_renderer.js

```diff
@@ -503,7 +503,12 @@ class Beyond20RollRenderer {
         const {rolls} = this.getToHit(request, "", data, type)
         await this._roller.resolveRolls(title, rolls, request);
         this.processToHitAdvantage(request.advantage, rolls);
-        return this.postDescription(request, title, null, {}, null, rolls);
+
+        const roll_info = [];
+        if (request["effects"] !== undefined && request["effects"].length !== 0)
```

Comment:
I'm thinking of a third party sending a roll and putting `request.effects = null` or even `request.effect = "Bless"` which would cause a crash here.

A safer check would be:
```js
if (Array.isArray(request["effects"]) && request["effects"].length > 0)
```